integration/confidential/fixtures/offline-fs-kbc/x86_64/aa-offline_fs_kbc-resources.json

```diff
@@ -1,5 +1,8 @@
 {
   "Policy": "ewogICAgImRlZmF1bHQiOiBbCiAgICAgICAgewogICAgICAgICAgICAidHlwZSI6ICJpbnNlY3VyZUFjY2VwdEFueXRoaW5nIgogICAgICAgIH0KICAgIF0sCiAgICAidHJhbnNwb3J0cyI6IHsKICAgICAgICAiZG9ja2VyIjogewogICAgICAgICAgICAicXVheS5pby9rYXRhLWNvbnRhaW5lcnMiOiBbCiAgICAgICAgICAgICAgICB7CiAgICAgICAgICAgICAgICAgICAgInR5cGUiOiAic2lnbmVkQnkiLAogICAgICAgICAgICAgICAgICAgICJrZXlUeXBlIjogIkdQR0tleXMiLAogICAgICAgICAgICAgICAgICAgICJrZXlQYXRoIjogIi9ldGMvY29udGFpbmVycy9xdWF5X3ZlcmlmaWNhdGlvbi9wdWJsaWMuZ3BnIgogICAgICAgICAgICAgICAgfQogICAgICAgICAgICBdCiAgICAgICAgfQogICAgfQp9Cg==",
```

Comment:
Also, we do not need `"Policy"`, `"Sigstore Config"` and `"GPG Keyring"` anymore